docs/micros/custom_domains.md

```diff
@@ -37,7 +37,8 @@ You need to consider the following when assigning a custom domain:
 - The time taken for the DNS record changes to take effect depends on your DNS settings and provider. It may take up to 24 hours for your domain to go live.
 - If you get an SSL error after the DNS record changes have propagated, please wait until the system has created an SSL certificate for your domain. This is usually fast but sometimes it may take a few hours.
 - If you have a [CAA record](#caa-records) set up for you domain, make sure you have set `sectigo.com` as an allowed CA for your domain. 
-- If you are adding a custom domain for the `www.` subdomain, **you need to add both `www.` and the main domain** in your dashboard. For eg, if you are setting up a custom domain for `example.com`, then add both `example.com` and `www.example.com` as custom domains for your micro.
+- If you are adding a custom domain for the `www.` subdomain, **you need to add both `www.` and the main domain** in your dashboard. For eg, if you are setting up a custom domain for `www.example.com`, then add both `example.com` and `www.example.com` as custom domains for your micro.
```

Comment:
- If you are adding a custom domain for the `www.` subdomain, **you need to add both `www.` and the main domain** in your dashboard. For example, if you are setting up a custom domain for `www.example.com`, then add both `example.com` and `www.example.com` as custom domains for your micro.